src/profiling/meter_manager.cpp

```diff
@@ -18,32 +16,34 @@ meter_manager::meter_manager() {
 };
 
 void meter_manager::checkpoint(std::string name) {
-    checkpoint_names_.push_back(std::move(name));
+    checkpoint_names.push_back(std::move(name));
 
     // Enforce a global synchronization point the first time that the meters
     // are used, to ensure that times measured across all domains are
     // synchronised.
-    if (meters_.size()==0) {
+    if (meters.size()==0) {
```

Comment:
Yes. And you would also be correct to point out that I still have not fixed the bug you found in the first review!